modeltranslation/manager.py

```diff
@@ -386,6 +371,24 @@ def update(self, **kwargs):
 
     update.alters_data = True
 
+    def _update(self, values):
+        """
+        This method is called in .save() method to update an existing record.
+        Here we force to update translation fields as well if the original
+        field only is passed in `save()` in argument `update_fields`.
+        """
+        field_names_to_update = {field.name for field, _, _ in values}
+
+        extra_values = []
+        for field, something, value in values:
```

Comment:
`something` is bad name, take a look at django source, to see what's really there and name it accordingly.

Reply:
Looks like it should be a `Type[Model]`, but it's always None